AmplifyPlugins/Geo/AWSLocationGeoPlugin/AWSLocationGeoPlugin+Configure.swift

```diff
@@ -32,21 +29,41 @@ extension AWSLocationGeoPlugin {
     public func configure(using configuration: AWSLocationGeoPluginConfiguration) throws {
         let authService = AWSAuthService()
         let credentialsProvider = authService.getCredentialsProvider()
-        let region = configuration.region
+        let region = configuration.regionName
+        let serviceConfiguration = try getLocationClientConfig(credentialsProvider: credentialsProvider, region: region)
 
-        let serviceConfiguration = AmplifyAWSServiceConfiguration(region: region,
-                                                                  credentialsProvider: credentialsProvider)
-        AWSLocation.register(with: serviceConfiguration,
-                             forKey: key)
-
-        let location = AWSLocation(forKey: key)
+        let location = LocationClient(config: serviceConfiguration)
         let locationService = AWSLocationAdapter(location: location)
 
         configure(locationService: locationService,
                   authService: authService,
                   pluginConfig: configuration)
     }
 
+    private func getLocationClientConfig(credentialsProvider: CredentialsProvider, region: String) throws -> LocationClient.LocationClientConfiguration {
```

Comment:
Our fix for the configuration issue in the SDK was released, so we can update to the latest version of the SDK and remove this workaround now.